openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -65,22 +68,22 @@ <h2 class="staff-info__title ui-staff__title">
                     </div>
 
                     <div class="staff-info__status ui-staff__content__section">
-                        <a aria-role="button" href="" id="schedule_training" class="action--submit"><span class="copy">{% trans "Schedule Example-Based Assessment Training" %}</span></a>
-                        <div id="schedule_training_message"></div>
+                        <button class="action--submit action--submit-training"><span class="copy">{% trans "Schedule Example-Based Assessment Training" %}</span></button>
```

Comment:
@andy-armstrong did you test that this schedule training stuff works now without the IDs? I'm not sure how to get into that scenario.